book.js

```diff
@@ -15,7 +19,19 @@ for (var i = 0; i < maxQuantity; i++) {
 }
 
 $quantity.onchange = function() {
-  $total.value = unitPrice * $quantity.value;
+  $subtotal.value = unitPrice * $quantity.value;
+  $total.value = Number($subtotal.value) + Number($shipping.value);
 };
 
 $quantity.onchange(null);
+
+$shippingCheckbox.onclick = function() {
+  if ($shippingCheckbox.checked) {
+    $shipping.value = intShippingPrice;
+  } else {
+    $shipping.value = 0;
+  }
+  $total.value = Number($subtotal.value) + Number($shipping.value);
+};
+
+$total.value = Number($subtotal.value) + Number($shipping.value);
```

Comment:
You are repeating this line 3 times. I think you can put this in a function called `updateTotalValue()` and then you just call that function multiple times.

```javascript
function updateTotal(){
    $total.value = Number($subtotal.value) + Number($shipping.value);
}
```